modules/distr-tracing-otel-remove-web-console.adoc

```diff
@@ -0,0 +1,26 @@
+//Module included in the following assemblies:
+//
+//* distr_tracing_install/dist-tracing-otel-removing.adoc
+
+:_content-type: PROCEDURE
+[id="distr-tracing-removing-otel-instance_{context}"]
+= Removing a {OTELShortName} instance by using the OpenShift web console
+
+You can remove a {OTELShortName} instance in the *Administrator* view of the OpenShift web console.
+
+.Prerequisites
+
+* You are logged in to the OpenShift web console as a cluster administrator with the `cluster-admin` role.
+
+.Procedure
+
+. Go to *Operators* -> *Installed Operators* -> *{OTELOperator}* -> *OpenTelemetryInstrumentation* or *OpenTelemetryCollector*.
++
+[TIP]
+====
+You can use the *Project:* drop-down list to select the relevant OpenShift project.
+====
+
+. Remove the relevant instance: go to {kebab} -> *Delete* ... -> *Delete*.
```

Comment:
Same comment here as I have left elsewhere. Won't keep repeating it, but suggest making the change across the board.

Reply:
Fixed.